apps/public-reference/pages/applications/household/add-members.tsx

```diff
@@ -73,25 +73,27 @@ export default () => {
           </h2>
         </div>
 
-        <div className="form-card__pager-row">
-          <form onSubmit={handleSubmit(onSubmit)}>
-            <div className={errors.householdSize ? "mb-8" : ""}>
-              <HouseholdSizeField
-                listing={listing}
-                householdSize={application.householdSize}
-                validate={true}
-                register={register}
-                error={errors.householdSize}
-                clearErrors={clearErrors}
-                assistanceUrl={t("application.household.assistanceUrl")}
-              />
-            </div>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <div className={errors.householdSize ? "" : ""}>
+            <HouseholdSizeField
+              listing={listing}
+              householdSize={application.householdSize}
+              validate={true}
```

Comment:
Totally pedantic, but `true` is assumed - you can just write `<HouseholdSizeField validate ...` (don't need to explicitly specify `validate={true}`)

Reply:
@bencpeters again, this is a mystery to me as well!